docs/API/content-type/listing-co.md

```diff
@@ -0,0 +1,61 @@
+title: How to list Content Objects | Flotiq docs
+description: How to list Content Objects in Flotiq
+
+
```

Comment:
I think this page is missing the way of listing content objects - an API endpoint for example and simple request